com.unity.shadergraph/Documentation~/surface-options.md

```diff
@@ -0,0 +1,38 @@
+# Modify surface options without changing your graph
+
+## Description
+
+Enable **Allow Material Override** to modify a specific set of properties for Universal Render Pipeline Lit and Unlit Shader Graphs and for Built-In Render Pipeline Shader Graphs in the Material Inspector.
+
+This functionality makes it possible for you to override the following Universal Render Pipeline Lit shader properties:
+
+* Workflow Mode
```

Comment:
I think this would work best as a table. Each entry could either have a link to the relevant page of documentation (where there is a matching exposed property in the Inspector), or a short description (where there is not a matching exposed property in the Inspector). We could then use the same format for the Built-in Render Pipeline.

Reply:
Cool